lampod/src/ln/channel_manager.rs

```diff
@@ -127,6 +127,15 @@ impl LampoChannelManager {
         self.handler.borrow().clone().unwrap()
     }
 
+    fn is_syncing(&self, block_event_height: u32) -> error::Result<bool> {
+        const SYNC_THRESHOLD: u32 = 5;
+
+        let (_, best_block_height) = self.onchain.backend.get_best_block()?;
```

Comment:
Hey @vincenzopalazzo 



Hey @vincenzopalazzo

I would appreciate a bit of guidance here.
cause the method get_best_block is not giving me the latest block height from the network